docs/agents/browser.md

```diff
@@ -0,0 +1,123 @@
+title: Browser SDK
+description: Ship browser metrics to Sematext Experience.
+
+Sematext [Browser SDK](https://github.com/sematext/browser-sdk) is an open-source framework used to ship web application metrics related to user interaction. Using [Browser SDK](https://github.com/sematext/browser-sdk) allows gathering metrics related to:
+ 
+ - Hard page loads for traditional web applications as well as the single-page applications
+ - Soft page loads generated via the `routeChange` command for the single-page applications
+ - HTTP requests
+ - [Web application resources, such as images, CSS files, etc.](/experience/resources/)
+ - [Web vitals](/experience/webvitals/)
+ - [Element timing](/experience/element-timing/)
+ - [Long tasks](/experience/longtasks/)
+ - [User sessions](/experience/user-identification/)
+ - [On-page transactions](/experience/on-page-transaction/)
+
+![Sematext Experience Overview](../images/experience/overview.png)
+
+## Development
+
+The RUM script is developed using [ECMAScript 2015](https://en.wikipedia.org/wiki/ECMAScript) and uses a few tools to help with the build and working with the code. Those tools include:
+ 
+ - `npm` - [Node package manager](https://npm.org)
+ - `yarn` - [Package dependency manager](https://yarnpkg.com)
+ - `flow` - [Static type checker](https://flow.org)
+ - `Cypress.io` - [Integration tests framework](https://www.cypress.io)
+
+### The Architecture of the RUM Script
+
+There are two parts of the RUM script: 
+
+ - The loader script,
+ - The metrics gathering script itself.
+
+The loader script is responsible for loading the main, metrics gathering script itself and adds the actual `<script>` tags with the content on the main metric script. 
```

Comment:
> The loader script is responsible for loading the main, metrics gathering script itself and adds.

The loader script is responsible for loading the metrics gathering script.  It adds ...

> on the main

Is this supposed to of OF the main?

main metric script ==> metrics gathering script (because that is what we are calling it in the list above)

Reply:
Adjusted, thanks.